xbmc/cores/RetroPlayer/rendering/RPRenderManager.cpp

```diff
@@ -177,7 +183,10 @@ void CRPRenderManager::FrameMove()
 
     if (m_state == RENDER_STATE::CONFIGURING)
     {
+      UpdateResolution();
```

Comment:
Currently this function is dead code. Its purpose is borrowed from VideoPlayer for adjusting the screen's refresh rate to match the video. Is this a setting that makes sense for games?